src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -20,12 +20,15 @@
 package ch.njol.skript.classes.data;
 
 import java.io.StreamCorruptedException;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 import java.util.Map.Entry;
 import java.util.UUID;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
```

Comment:
Do you need these imports?

Reply:
Yeah

Reply:
From which line of code...?

Reply:
Nevermind, they were needed in the old code before I used EnumUtils and I forgot to remove them, my bad